Source/ARTDeviceIdentityTokenDetails.m

```diff
@@ -72,34 +73,11 @@ + (BOOL)supportsSecureCoding {
 #pragma mark - Archive/Unarchive
 
 - (NSData *)archive {
-    if (@available(macOS 10.13, iOS 11, tvOS 11, *)) {
-        NSError *error;
-        NSData *data = [NSKeyedArchiver archivedDataWithRootObject:self requiringSecureCoding:false error:&error];
-        if (error) {
-            NSLog(@"ARTDeviceIdentityTokenDetails Archive failed: %@", error);
-        }
-        return data;
-    }
-    else {
-        return [NSKeyedArchiver archivedDataWithRootObject:self];
-    }
+    return [self art_archive];
```

Comment:
We don't use the "snake_notation" if it's possible please change it to the "camelCase".

Reply:
I was just following notation already presented in this file. And it's "snake". I think we should either rename all at once or just leave it as is (in this file at least).